### ISSUE TITLE
Models changes not being detected in 1.1.3

### ISSUE THREAD

**Issue by Benoit1980** [TiagoSilvaPereira/vemto-issues#305]
Hi Tiago,

I have installed Paddle from the Laravel package:

```
use Illuminate\Database\Eloquent\Model;
use Laravel\Paddle\Billable;
 
class Team extends Model
{
    use Billable;
}
```

I had to add the above code to the user model for it to work. Upon regenerating the code, the model updates have been erased. The .ENV changes and app.blade.php have been detected but not the USER model changes.

Thanks,

Ben

**Comment by TiagoSilvaPereira** [comment]
Hi @Benoit1980 thank you for reporting... I tested it here and can confirm, but this problem happens only with the **User** model... I'll fix it soon

**Comment by Benoit1980** [comment]
Thanks @TiagoSilvaPereira :-)

**Comment by TiagoSilvaPereira** [comment]
Fixed on 1.1.5
